frontend/src/app/libs/common-components-demo/slide-toggle-demo/slide-toggle-demo.component.html

```diff
@@ -0,0 +1,13 @@
+<mat-card
```

Comment:
Please use app-demo-group with 2 divs for light-theme and dark theme inside. Also use app-demo-state for each component's state.
Check out event-tile-demo.component.html with example.